pkg/network/protocols/http/monitor.go

```diff
@@ -248,9 +249,8 @@ func (m *Monitor) GetUSMStats() map[string]interface{} {
 	}
 
 	if m != nil {
-		response["last_check"] = m.httpTelemetry.then
+		response["last_check"] = m.httpTelemetry.lastCheck
```

Comment:
Does this need a `Load()` (or equivalent)? I think `lastCheck` is a `*atomic.Int64`